src/main/java/io/confluent/connect/hdfs/DateTimeUtils.java

```diff
@@ -0,0 +1,13 @@
+package io.confluent.connect.hdfs;
+
+import org.joda.time.DateTimeZone;
+
+public class DateTimeUtils {
+    private static long DAY_IN_MS = 24 * 60 * 60 * 1000;
+    public static long getNextTimeAdjustedByDay(long currentTimeMs, long periodMs, DateTimeZone timeZone) {
```

Comment:
Would be nice to add some documentation on what this utils is trying to achieve.